relative_wallclock/relative_wallclock.py

```diff
@@ -0,0 +1,48 @@
+"""
+Adds an info-wallclock-relative command which prints the approximate wallclock time relative to the
+start of recording.
+    Usage: info-wallclock-relative
+"""
+
+import datetime
+import re
+
+import gdb
+
+from undo.debugger_extensions import (
+    debugger_utils,
+    udb,
+)
+
+
+class WallclockRelative(gdb.Command):
+    def __init__(self):
+        super().__init__("info-wallclock-relative", gdb.COMMAND_USER)
+
+    def invoke(self, arg, from_tty):
+        extents = debugger_utils.execute_to_string("info wallclock-extent")
+        m = re.search("Start time: (.*)", extents)
+        if m:
```

Comment:
This could be more readable with left nesting having an early return. And maybe some appropriate error message:

```
if m is None:
    raise gdb.GdbError("The start time of the execution history is not available.")
```